frontend/apps/reader/modules/readergesture.lua

```diff
@@ -352,6 +360,41 @@ function ReaderGesture:addToMainMenu(menu_items)
             },
         },
     }
+    local double_tap_submenu = {
+        text = _("Double tap"),
+        enabled_func = function()
+            return self.ges_mode == "gesture_reader" and self.ui.disable_double_tap ~= true
+        end,
+        sub_item_table = {
+            {
+                text_func = function() return actionTextFunc("double_tap_left_side", _("Left side")) end,
+                enabled_func = function() return self.ges_mode == "gesture_reader" end,
+                sub_item_table = self:buildMenu("double_tap_left_side", self.default_gesture["double_tap_left_side"]),
+            },
+            {
+                text_func = function() return actionTextFunc("double_tap_right_side", _("Right side")) end,
+                sub_item_table = self:buildMenu("double_tap_right_side", self.default_gesture["double_tap_right_side"]),
+            },
+            {
+                text_func = function() return actionTextFunc("double_tap_top_left_corner", _("Top left")) end,
+                enabled_func = function() return self.ges_mode == "gesture_reader" end,
```

Comment:
Why does that top left one has this enabled_func, and not the others?

Reply:
Copy paste oversight. Top left (tap) isn't implemented in the gesture manager in the file manager.